### DISCUSSION TITLE
My GitHub account has been "flagged" and is hidden from public view

### DISCUSSION THREAD

**Discussion by ghost** [community/community#45760] [Other Feature Feedback, Questions, & Ideas] [answered]
Resolved by support

**Comment by ghost** [comment, marked as answer]
Resolved by support

**Reply by LazyKoalaMan** [reply]
How would you contact them? Through mail or call?

**Reply by ghost** [reply]
> How would you contact them? Through mail or call?

You can contact to GitHub's Support Team through the "My Tickets" page.

**Reply by ghost** [reply]
Open a Ticket: https://support.github.com/tickets/personal prepare to wait 4-6 Weeks -.-

**Comment by mstangurak** [comment]
Same issue happened to me. My account has been flagged as well and I didn't do anything wrong. How to contact support?

**Comment by YossefMohamed** [comment]
what did you do to solve this problem
as my account has been flagged also

**Comment by maheshachinnaswamy1994** [comment]
My GitHub account also been flagged. This is completely weird as there is no intimation from GitHub about the same. I guess this is one of the reason why people are moving towards other VCS.

**Comment by PraveenVerma17** [comment]
My account is flagged, could you please unflag my account.
